src/DataViews/SamplingReportDataView.cpp

```diff
@@ -343,39 +342,73 @@ void SamplingReportDataView::SetStackEventsCount(uint32_t stack_events_count) {
-    count_type = "inclusive";
-    at_the_top_or_encountered = "encountered";
-  } else {
-    raw_count = function.exclusive;
-    percentage = function.exclusive_percent;
-    count_type = "exclusive";
-    at_the_top_or_encountered = "at the top of the callstack";
-  }
-
-  orbit_statistics::BinomialConfidenceInterval interval =
-      app_->GetConfidenceIntervalEstimator().Estimate(percentage / 100.0f, stack_events_count_);
-
+[[nodiscard]] static std::string BuildTooltipTail(
+    uint32_t stack_events_count, float percentage,
+    const orbit_statistics::BinomialConfidenceInterval& interval) {
   return absl::StrFormat(
-      "The function \"%s\"\n"
-      "was %s %u times (%s count)\n"
       "in a total of %u stack samples.\n"
       "This makes up for %.2f%% of samples.\n\n"
       "The 95%% confidence interval for the true percentage is\n"
       "(%.2f%%, %.2f%%).",
-      function.name, at_the_top_or_encountered, raw_count, count_type, stack_events_count_,
-      percentage, interval.lower * 100.0f, interval.upper * 100.0f);
+      stack_events_count, percentage, interval.lower * 100.0f, interval.upper * 100.0f);
+}
+
+[[nodiscard]] std::string SamplingReportDataView::BuildToolTipInclusive(
+    const SampledFunction& function) const {
+  const orbit_statistics::BinomialConfidenceInterval interval =
+      app_->GetConfidenceIntervalEstimator().Estimate(function.inclusive_percent / 100.0f,
+                                                      stack_events_count_);
+  const std::string head = absl::StrFormat(
+      "The function \"%s\"\n"
+      "was encountered %u times (inclusive count)\n",
+      function.name, function.inclusive);
+  return absl::StrCat(head,
+                      BuildTooltipTail(stack_events_count_, function.inclusive_percent, interval));
+}
+
+[[nodiscard]] std::string SamplingReportDataView::BuildToolTipExclusive(
+    const SampledFunction& function) const {
+  const orbit_statistics::BinomialConfidenceInterval interval =
+      app_->GetConfidenceIntervalEstimator().Estimate(function.exclusive_percent / 100.0f,
+                                                      stack_events_count_);
+
+  const std::string head = absl::StrFormat(
+      "The function \"%s\"\n"
+      "was at the top of the callstack %u times (exclusive count)\n",
+      function.name, function.exclusive);
+
+  return absl::StrCat(head,
+                      BuildTooltipTail(stack_events_count_, function.exclusive_percent, interval));
+}
+
+[[nodiscard]] std::string SamplingReportDataView::BuildToolTipUnwindErrors(
+    const SampledFunction& function) const {
+  if (function.unwind_errors == 0) return "";
+
+  const orbit_statistics::BinomialConfidenceInterval interval =
+      app_->GetConfidenceIntervalEstimator().Estimate(function.unwind_errors_percent / 100.0f,
+                                                      stack_events_count_);
+  const std::string head = absl::StrFormat(
+      "%u samples with the function \"%s\" \n"
```

Comment:
```suggestion
      "%u samples with the function \"%s\"\n"
```
It's not there in the other tooltips.